server/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -261,7 +275,7 @@ private TranslogConfig getTranslogConfig(final Path path, final Settings setting
         );
 
         final IndexSettings indexSettings = IndexSettingsModule.newIndexSettings(shardId.getIndex(), settings);
-        return new TranslogConfig(shardId, path, indexSettings, NON_RECYCLING_INSTANCE, bufferSize);
+        return new TranslogConfig(shardId, path, indexSettings, NON_RECYCLING_INSTANCE, bufferSize, RANDOMIZING_IO_BUFFERS);
```

Comment:
Can we randomly use `DiskIoBufferPool.INSTANCE` too?

Reply:
++ done